src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/IPInterfacePropertiesTest_Linux.cs

```diff
@@ -21,207 +21,225 @@ public IPInterfacePropertiesTest_Linux()
+                    _log.WriteLine("- Multicast Addresses: " + ipProperties.MulticastAddresses.Count);
+                    foreach (IPAddressInformation multi in ipProperties.MulticastAddresses)
+                    {
+                        _log.WriteLine("-- " + multi.Address.ToString());
+                        Assert.Throws<PlatformNotSupportedException>(() => multi.IsDnsEligible);
+                        Assert.Throws<PlatformNotSupportedException>(() => multi.IsTransient);
+                    }
+
+                    Assert.NotNull(ipProperties.UnicastAddresses);
+                    _log.WriteLine("- Unicast Addresses: " + ipProperties.UnicastAddresses.Count);
+                    foreach (UnicastIPAddressInformation uni in ipProperties.UnicastAddresses)
+                    {
+                        _log.WriteLine("-- " + uni.Address.ToString());
+                        Assert.Throws<PlatformNotSupportedException>(() => uni.AddressPreferredLifetime);
+                        Assert.Throws<PlatformNotSupportedException>(() => uni.AddressValidLifetime);
+                        Assert.Throws<PlatformNotSupportedException>(() => uni.DhcpLeaseLifetime);
+                        Assert.Throws<PlatformNotSupportedException>(() => uni.DuplicateAddressDetectionState);
+
+                        Assert.NotNull(uni.IPv4Mask);
+                        _log.WriteLine("--- IPv4 Mask: " + uni.IPv4Mask);
+                        Assert.Throws<PlatformNotSupportedException>(() => uni.IsDnsEligible);
+                        Assert.Throws<PlatformNotSupportedException>(() => uni.IsTransient);
+                        Assert.Throws<PlatformNotSupportedException>(() => uni.PrefixOrigin);
+                        Assert.Throws<PlatformNotSupportedException>(() => uni.SuffixOrigin);
+
+                        // Prefix Length
+                        _log.WriteLine("--- PrefixLength: " + uni.PrefixLength);
+                        Assert.True(uni.PrefixLength > 0);
+                        Assert.True((uni.Address.AddressFamily == AddressFamily.InterNetwork ? 33 : 129) > uni.PrefixLength);
+                    }
+
+                    Assert.NotNull(ipProperties.WinsServersAddresses);
+                    _log.WriteLine("- Wins Addresses: " + ipProperties.WinsServersAddresses.Count);
+                    foreach (IPAddress wins in ipProperties.WinsServersAddresses)
+                    {
+                        _log.WriteLine("-- " + wins.ToString());
+                    }
                 }
-
-                Assert.NotNull(ipProperties.WinsServersAddresses);
-                _log.WriteLine("- Wins Addresses: " + ipProperties.WinsServersAddresses.Count);
-                foreach (IPAddress wins in ipProperties.WinsServersAddresses)
-                {
-                    _log.WriteLine("-- " + wins.ToString());
-                }
-            }
+            }).WaitAsync(TimeSpan.FromSeconds(30));
```

Comment:
In some places we use something like `TestHelper.PassingTestTimeout`. Maybe put it to `Common` or create const instead of using number directly?

Reply:
Added link to Common.